functions/model/milestone/lib.js

```diff
@@ -73,7 +73,7 @@ exports.getMilestoneData = function(orgDomain, milestoneId) {
  * @return {any}
  */
 exports.getAllMilestonesData = function(orgDomain, teamId="") {
-  let query = db.collection("Organizations").doc(orgDomain).collection("Milestones");
+  let query = db.collection("Organizations").doc(orgDomain).collection("Milestones").where("MilestoneStatus", "==", "Deleted");
   if (teamId != "") {
     query = query.where("TeamId", "==", teamId);
```

Comment:
This will get the deleted Milestones is this the required functionality?

Reply:
Actually it is used to fetch the milestone but the condition here is wrong , it should be not equal to "Deleted", I have made the changes in the code as required.